addons/columns/style.css

```diff
@@ -20,7 +20,7 @@
 .scratchCategoryItemBubble,
 .scratchCategoryItemIcon {
   margin: 0;
-  margin-right: 0.5rem;
+  margin-inline-end: 0.5rem;
```

Comment:
Editor addons typically don't use these inline-start/end etc. CSS properties, what do you think @mxmou?

Reply:
mxmou *did* that xd 

https://github.com/ScratchAddons/ScratchAddons/pull/3491/commits/4b880f96b6d5755459c850757dda0e96427d6187

Reply:
lmao

Reply:
None of these horizontal paddings and margins override Scratch's ones, so using logical properties is fine (and makes the code shorter).